src/components/Navbar/Navbar.js

```diff
@@ -1,6 +1,17 @@
 import React from "react";
 import handLogo from "../../assets/hand.png";
+import { useAuth } from "../../Context/auth-context";
+import { Link, useNavigate } from "react-router-dom";
+
 export default function Navbar() {
+  const { authState, setAuthState } = useAuth();
```

Comment:
You can destructure it as:
```
  const { authState:{isUserLoggedIn}, setAuthState } = useAuth();
```

Reply:
Not found it much readable.so, previous one is more preferable